ckanext/saml2/model/user_sso_gen.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, UnicodeText, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+import ckan.model as model
+
+Base = declarative_base()
+metadata = Base.metadata
+
+
+class UserSsoGen(Base):
+    __tablename__ = 'saml2_user_sso_gen'
+
+    id = Column(UnicodeText, primary_key=True)
+    gen = Column(UnicodeText, nullable=False, unique=True)
+    user_name = Column(UnicodeText, nullable=False, unique=True)
+    allow_update = Column(Boolean)
+    state = Column(UnicodeText, default='active')
```

Comment:
Why is there an additional state to the existing state in the user table? This seems to risk causing confusion about what the real state of the user is.